messenger.go

```diff
@@ -152,7 +154,7 @@ func (m *Messenger) Handler() http.Handler {
 }
 
 // ProfileByID retrieves the Facebook user associated with that ID
```

Comment:
Should add a note to this doc-comment that providing no arguments will retrieve the defaults (instead of nothing, as you may expect).

Reply:
👍